src/test/java/com/codeborne/selenide/collections/ExactTextsTest.java

```diff
@@ -110,15 +110,12 @@ void failOnTextSizeMismatch() {
     WebElementsCollection mockedElementsCollection = mock(WebElementsCollection.class);
     when(mockedElementsCollection.description()).thenReturn("Collection description");
 
-    try {
-      exactTexts.fail(mockedElementsCollection,
-        singletonList(mockedWebElement),
-        exception,
-        10000);
-    } catch (TextsSizeMismatch ex) {
-      assertThat(ex)
-        .hasMessage("\nActual: [One, Two], List size: 1\nExpected: [One, Two], List size: 2\nCollection: Collection description");
-    }
+    assertThatThrownBy(() -> exactTexts.fail(mockedElementsCollection,
+      singletonList(mockedWebElement), exception, 10000))
+      .isInstanceOf(TextsSizeMismatch.class)
+      .hasMessageContaining("Actual: [One, Two], List size: 1")
```

Comment:
Maybe better to use `hasMessageStartingWith` in this case?

Reply:
There is System.getLineSeparator() in the beginning. 
Do you think that really meaningfull here? It can worsen readability.

Reply:
oh. correct! let's go with current assertion. thanks